ambry-store/src/test/java/com.github.ambry.store/StoreFindTokenTest.java

```diff
@@ -91,13 +92,22 @@ public void equalityTest() {
     compareTokens(journalToken, otherJournalToken);
     compareTokens(inclusiveJournalToken, otherInclusiveJournalToken);
 
+    UUID newSessionId = sessionId;
```

Comment:
Minor: this logic for generating a different UUID could go into a small helper function